### ISSUE TITLE
Install on Windows 10?

### ISSUE THREAD

**Issue by hulsmeier** [tyagi-iiitv/PointPillars#6]
I'm running python setup.py install on Windows 10 but when i try to run training it tells me "No Module named 'point_pillars'. Is Windows not supported or am I missing something?

**Comment by tyagi-iiitv** [comment]
Yes, some people have faced this issue before. I think this happens because the PointPillars.cpp file is installed at a path that is not readable with python. 

If you look at the logs from `python setup.py install`, the files are being copied to build/bdist.linux-x86_64/egg. This directory in your case is not readable. Maybe you can copy the installed files and paste them into your Point Pillars directory, that might work. 

```
creating build/bdist.linux-x86_64/egg
copying build/lib.linux-x86_64-3.6/point_pillars.cpython-36m-x86_64-linux-gnu.so -> build/bdist.linux-x86_64/egg
creating stub loader for point_pillars.cpython-36m-x86_64-linux-gnu.so
byte-compiling build/bdist.linux-x86_64/egg/point_pillars.py to point_pillars.cpython-36.pyc
creating build/bdist.linux-x86_64/egg/EGG-INFO
copying point_pillars.egg-info/PKG-INFO -> build/bdist.linux-x86_64/egg/EGG-INFO
copying point_pillars.egg-info/SOURCES.txt -> build/bdist.linux-x86_64/egg/EGG-INFO
copying point_pillars.egg-info/dependency_links.txt -> build/bdist.linux-x86_64/egg/EGG-INFO
copying point_pillars.egg-info/not-zip-safe -> build/bdist.linux-x86_64/egg/EGG-INFO
copying point_pillars.egg-info/top_level.txt -> build/bdist.linux-x86_64/egg/EGG-INFO
writing build/bdist.linux-x86_64/egg/EGG-INFO/native_libs.txt
creating 'dist/point_pillars-0.0.1-py3.6-linux-x86_64.egg' and adding 'build/bdist.linux-x86_64/egg' to it
```

**Comment by hulsmeier** [comment]
So I tried copying those files but that didn't work. Here is my log.

running install
running bdist_egg
running egg_info
writing point_pillars.egg-info\PKG-INFO
writing dependency_links to point_pillars.egg-info\dependency_links.txt
writing top-level names to point_pillars.egg-info\top_level.txt
reading manifest file 'point_pillars.egg-info\SOURCES.txt'
writing manifest file 'point_pillars.egg-info\SOURCES.txt'
installing library code to build\bdist.win-amd64\egg
running install_lib
running build_ext
-- Selecting Windows SDK version 10.0.18362.0 to target Windows 10.0.20190.
-- pybind11 v2.6.0 dev
-- Configuring done
-- Generating done
-- Build files have been written to: D:/Documents/GitHub/PointPillars/build/temp.win-amd64-3.6/Release
Microsoft (R) Build Engine version 16.6.0+5ff7b0c9e for .NET Framework
Copyright (C) Microsoft Corporation. All rights reserved.

  point_pillars.vcxproj -> D:\Documents\GitHub\PointPillars\build\temp.win-amd64-3.6\Release\Release\point_pillars.cp36
  -win_amd64.pyd
warning: install_lib: 'build\lib.win-amd64-3.6' does not exist -- no Python modules to install

creating build\bdist.win-amd64\egg
creating build\bdist.win-amd64\egg\EGG-INFO
copying point_pillars.egg-info\PKG-INFO -> build\bdist.win-amd64\egg\EGG-INFO
copying point_pillars.egg-info\SOURCES.txt -> build\bdist.win-amd64\egg\EGG-INFO
copying point_pillars.egg-info\dependency_links.txt -> build\bdist.win-amd64\egg\EGG-INFO
copying point_pillars.egg-info\not-zip-safe -> build\bdist.win-amd64\egg\EGG-INFO
copying point_pillars.egg-info\top_level.txt -> build\bdist.win-amd64\egg\EGG-INFO
creating 'dist\point_pillars-0.0.1-py3.6-win-amd64.egg' and adding 'build\bdist.win-amd64\egg' to it
removing 'build\bdist.win-amd64\egg' (and everything under it)
Processing point_pillars-0.0.1-py3.6-win-amd64.egg
removing 'c:\users\oneal\appdata\local\programs\python\python36\lib\site-packages\point_pillars-0.0.1-py3.6-win-amd64.egg' (and everything under it)
creating c:\users\oneal\appdata\local\programs\python\python36\lib\site-packages\point_pillars-0.0.1-py3.6-win-amd64.egg
Extracting point_pillars-0.0.1-py3.6-win-amd64.egg to c:\users\oneal\appdata\local\programs\python\python36\lib\site-packages
point-pillars 0.0.1 is already the active version in easy-install.pth

Installed c:\users\oneal\appdata\local\programs\python\python36\lib\site-packages\point_pillars-0.0.1-py3.6-win-amd64.egg
Processing dependencies for point-pillars==0.0.1
Finished processing dependencies for point-pillars==0.0.1

**Comment by tyagi-iiitv** [comment]
One way out that I can see is to run this code using a terminal in Jupyterlab. In that case, since you are running a conda environment, the dependencies will be installed in the correct directories. Can you try that? Try installing Anaconda for Python 3 on Windows.

**Comment by hulsmeier** [comment]
Hey so i tried your suggestion. Ran Jupyter Lab through Anaconda and then tried running the setup in a terminal. Still gives me the same problem though.

Is it something to do with this line?

warning: install_lib: 'build\lib.win-amd64-3.6' does not exist -- no Python modules to install

**Comment by tyagi-iiitv** [comment]
Hi just wanted to follow up on this, were you able to figure it out?

**Comment by hulsmeier** [comment]
Hi, I haven't been able to solve it unfortunately.

**Comment by tyagi-iiitv** [comment]
Ohh okay, I'll try it out on my system and will let you know if I figure out the soln.

**Comment by ma7555** [comment]
Were you able to figure it out??

**Comment by ma7555** [comment]
I was able to fix it on Windows, will fork and fix it as soon as possible

**Comment by tyagi-iiitv** [comment]
Hi @ma7555, can you please submit a PR with this fix? Thanks for the work!

**Comment by ma7555** [comment]
Hello @tyagi-iiitv, I didn't have the chance to check the solution on Linux/Ubuntu though...

**Comment by ma7555** [comment]
Hello again,

There is an easier solution that I found without altering the source code for now.

After `python setup.py install`

copy '.pyd' file from
`.\PointPillars\build\temp.win32-3.8\Release\Release` to `C:\Users\USERNAME\anaconda3\envs\ENVNAME\Lib\site-packages`
